examples/celery-weblate.service

```diff
@@ -13,14 +13,17 @@ ExecStartPre=/bin/mkdir -p /var/run/celery
 ExecStartPre=/bin/chown -R weblate /var/run/celery/
 ExecStartPre=/bin/mkdir -p /var/log/celery
 ExecStartPre=/bin/chown -R weblate /var/log/celery/
-ExecStart=/bin/sh -c '${CELERY_BIN} multi start ${CELERYD_NODES} \
+
+# please note surrounding environment variables with braces has a specific meaning.
+# When surrounded by braces, variable content is not split around spaces.
+ExecStart=${CELERY_BIN} multi start $CELERYD_NODES \
```

Comment:
I'm not really about using `${CELERY_BIN}`. In some setups I use `python3 -m celery` as `CELERY_BIN` and that would probably break it.

Reply:
Fixed in 2af051cd8bd449e80401d5f71a9bec7600a8b3bc